metrics/ci_integrations/docs/diagrams/config_validator_destination_class_diagram.puml

```diff
@@ -56,13 +51,17 @@
                     + results: UnmodifiableMapView<ConfigField, FieldValidationResult>
                 }
 
-                class FieldValidationResult {
+                class FieldValidationResult<T> {
```

Comment:
It seems like we've missed the relation between this class and `CoolIntegrationDestinationValidationDelegate` class.